DailyFeed/📱Controllers/NewsDetailViewController.swift

```diff
@@ -239,7 +239,7 @@ class NewsDetailViewController: UIViewController, SFSafariViewControllerDelegate
     // Helper method to open articles in Safari
     func openInSafari() {
         guard let articleString = articleStringURL, let url = URL(string: articleString) else { return }
-        let svc = DFSafariViewController(url: url)
+        let svc = SFSafariViewController(url: url)
```

Comment:
@lscuderi can you add `svc.preferredControlTintColor = .black` after this line?
black matches the colour scheme of the app